docs/VnVPlan/VnVPlan.tex

```diff
@@ -549,15 +549,330 @@ \subsection{Tests for Nonfunctional Requirements}
+is to have someone use the application and provide feedback.
+
+\begin{description}
+  \item[User Test Demo:] We will engage a user to test the application, bringing
+  them through a new user process, and ask for feedback on specific areas
+  relevant to the requirements that were set.
+\end{description}
+
+\paragraph{Validate Usability and Humanity tests}
+\begin{enumerate}
+
+  \item{test-UH-1}
+  
+  NFR: UH-E1, UH-E3, UH-L1, UH-UP1, UH-UP2, OE-P1
+  
+  Type: Manual, User Test Demo
+            
+  Initial State: Application on home page, ready for use
+            
+  Input/Condition: User engagement
+            
+  Output/Result: Subjective feedback for how well application scores on different requirements relating to user experience
+            
+  How test will be performed: 
+  \begin{enumerate}
+    \item A user will use the application, going through the onboarding process
+    and doing a demo of the application
+    \item After a full demo, the user will be asked to provide feedback on
+    different aspects of usability, such as ease of use, app clarity, and prompt
+    politeness.
+  \end{enumerate}
+            
+  \item{test-UH-2}
+  
+  NFR: UH-E2
+  
+  Type: Manual
+            
+  Initial State: Application on home page, ready for use
+            
+  Input: Tester engagement
+            
+  Output: A count for the number of clicks required to navigate to different functionality
+            
+  How test will be performed: 
+  \begin{enumerate}
+    \item A list of commonly used functions will be created (System is not large, therefore this can be done manually)
+    \item The user will navigate to each of those functions, and note how many clicks were required to get there
+  \end{enumerate}
+
+  \item{test-UH-3}
+  
+  NFR: UH-PI2
+
+  Type: Manual
+            
+  Initial State: Application on home page, ready for use
+            
+  Input: Tester engagement
+            
+  Output: All identified instances of foreign languages used
+            
+  How test will be performed: 
+  \begin{enumerate}
+    \item The tester will look through each page, and test functionality
+    \item They will verify that all pages and prompts are in the correct language (English (US))
+  \end{enumerate}
+  
+  \end{enumerate}
+
+\subsubsection{Versioning}
+These requirements are for versioning, in order to roll back to old iterations
+if needed. These will mainly be tested by checking if old versions are saved.
+Note that this check will be done after each iteration of the model
+
+\paragraph{Validate old versions of project}
+\begin{enumerate}
+  \item{test-V-1}
+
+  NFR: MS-M1, OR-R2
+  
+  Type: Manual
+            
+  Initial State: N/A
+            
+  Input/Condition: Tester engagement
+            
+  Output/Result: The way to find old iterations/versions of project
+            
+  How test will be performed: 
+  \begin{enumerate}
+    \item A tester must check if old iterations are stored properly (on Github, for example)
```

Comment:
Old iteration of what? We should store old models with appropriate semantic versioning.